### PR TITLE
Improving test failure logging

### PR DESCRIPTION
**Pull request by hello-fri-end**
https://trello.com/c/JZUQpvQc/1144-improve-test-failure-log-for-frontend-testing

### CODE REVIEW

**Review by CatB1t — CHANGES_REQUESTED** [review]
Hello @hello-fri-end, great work! I'd suggest adding a flag to enable or disable the full trace back.
for example `--tb_mode=none` or `--tb_mode=ivy`. also let's keep the default behaviour of the traceback to `--tb_mode=ivy` for now. Thank you!

**Comment by hello-fri-end** [comment]
> I'd suggest adding a flag to enable or disable the full trace back.

Thanks for the review! Pytest has an inbuilt flag `tb` for choosing between various traceback modes ( 'auto', 'native', 'long', 'short', 'line'). I added a flag `tb-ivy` for choosing between different tracebacks modes for ivy with the default value of `none`.

**Review by CatB1t — APPROVED** [review]
lgtm! great work @hello-fri-end, I've modified the traceback flag name and changed the default to be full, just to not cause issues with the CI, thank you!

File: ivy_tests/test_ivy/conftest.py [hunk 1]
```diff
@@ -39,14 +39,12 @@
...
-    if raw_value == "auto":
-        config.option.tbstyle = "line"
-    else:
-        config.option.tbstyle = raw_value
+
+    # Ivy Exception traceback
+    set_exception_trace_mode(config.getoption("--ivy-tb"))
+
+    # Pytest traceback
+    config.option.tbstyle = config.getoption("--tb")
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/conftest.py:47]: Hey @CatB1t! If we're going to keep the default as 'full', this line is not required. The default tracemode for Pytest is "auto" which displays the full traceback. I had actually changed the default behavior to "line" which displays a shorter, more concise traceback.

**Pull request merged**